### PR TITLE
show request type in reference sidebar

### PR DESCRIPTION
**Pull request by cameronrohani**
Before:
image

After:
image

### CODE REVIEW

File: packages/api-reference/src/components/SidebarElement.vue [hunk 1]
```diff
@@ -94,6 +95,9 @@ defineExpose({ el })
           class="sidebar-icon"
           :src="item.icon.src" />
         <p>{{ item.title }}</p>
+        <div
+          class="sidebar-heading-type"
+          :class="`sidebar-heading-type--${item.httpVerb}`"></div>
```
Comment (hanspagel) [inline packages/api-reference/src/components/SidebarElement.vue:100]: Am on mobile, maybe I’ve overseen it, but should we add this class only when httpVerb is set?

Comment (cameronrohani) [reply packages/api-reference/src/components/SidebarElement.vue:100]: 💯% 

Will update

**Comment by thelinuxlich** [comment]
nice!

**Review by marclave — APPROVED** [review]
great work @cameronrohani

**Pull request merged**